themes.json

```diff
@@ -205,7 +205,14 @@
       "pypi": "sphinx-nameko-theme"
     },
     {
-      "config": "sphinx_nefertiti",
+      "config": {
+        "html_theme": "sphinx_nefertiti",
+        "html_theme_options": {
+          "documentation_font": "Open Sans",
+          "monospace_font": "Ubuntu Mono",
+          "monospace_font_size": "1.1rem"
+        }
```

Comment:
I am not sure I understand you, but I put those changes back.

I have a question regarding theme source code updates. How do we, maintainers of Sphinx themes, request that project here updates the version of a theme? ie: I would like that the version of sphinx-nefertiti used to build the sample-site is 0.2.1. Is there a config file where we could modify themes versions?

Reply:
> I am not sure I understand you, but I put those changes back.

Thanks! I'll try rephrasing: It's nicer to have the examples use the defaults from the theme instead of overriding things like fonts, since that better reflects the default configuration for the theme (which is what a documentation author would get, if they change their `html_theme` over).

Reply:
Indeed, I do agree.